### PR TITLE
Add info about all types of fields supported by StorIO Annotation Processor

### LINKED ISSUE #497: Add info about supported field types (Annotation Processor)
http://stackoverflow.com/questions/32259956/what-are-the-various-column-types-you-can-have-in-storio

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #497.

@nikitin-da PTAL :)

### CODE REVIEW

File: docs/StorIOSQLite.md [hunk 1]
```diff
@@ -328,6 +328,8 @@ StorIOSQLite storIOSQLite = DefaultStorIOSQLite.builder()
   .build(); // This instance of StorIOSQLite will know how to work with Tweet objects
 ```
 
+BTW: [Here is a class](storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/entities/AllSupportedTypes.java) with all types of fields, supported by StorIO SQLite Annotation Processor.
+
```
Comment (nikitin-da) [inline docs/StorIOSQLite.md]: the link doesn't work from this directory

Comment (artem-zinnatullin) [inline docs/StorIOSQLite.md]: Ah, sure, my bad

Comment (artem-zinnatullin) [inline docs/StorIOSQLite.md]: Fixed!

**Comment by nikitin-da** [comment]
Great!

**Pull request merged**